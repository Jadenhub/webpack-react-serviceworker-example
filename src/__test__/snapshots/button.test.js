import renderer from 'react-test-renderer';
import React from 'react';
import Button from '../../components/interact/Button';

test('renders correctly', () => {
  const tree = renderer
    .create(<Button 
        lineHeight={2}
        padding={'4px'}
        disabled={true}
        border={'solid 1px #797979'}
      />
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});