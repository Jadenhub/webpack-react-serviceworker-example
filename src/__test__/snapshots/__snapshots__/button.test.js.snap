// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<button
  className="Button__ButtonWrapper-sc-7gm9an-0 dVbGaX"
  disabled={true}
/>
`;
